src/krux/pages/home.py

```diff
@@ -54,11 +56,82 @@ def __init__(self, ctx):
 
     def mnemonic(self):
         """Handler for the 'mnemonic' menu item"""
+        submenu = Menu(
+            self.ctx,
+            [
+                (t("Words"), self.display_mnemonic_words),
```

Comment:
How about:
- Words
- Plaintext QR
- Compact SeedQR
- Back

The "Compact SeedQR" option could then just be the transcribe version by default (it's still scannable, right?), and at the end the print prompt could appear.

In other words:
1. remove `display_compact_qr`
2. add `self.print_qr_prompt(code, FORMAT_NONE)` to the end of `transcribe_compact_qr`
3. rename `transcribe_compact_qr` to `display_compact_qr`